docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,10 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+- **DAML Standard Library**: Move the ``Tuple`` and ``Either`` types to ``daml-prim:DA.Types`` rather
+  than exposing internal locations. People using the Scala/Java bindings will need to change from
```

Comment:
Can you rephrase as an instruction? Just to make it a bit more direct, so it's totally obvious what you need to do. Something like "If you're using the Scala or Java bindings, you need to change..." and then what and where - is this in your imports, or what?

Reply:
and this doesn't affect your DAML code/imports at all?

Reply:
That would be great, but I have very little idea what you actually need to do.

Reply:
No impact on DAML imports at all - they were always being reexposed in other places and they still are

Reply:
If you don't know what users need to do as a result of this change, can you find out, or is that too much of a pain?

Reply:
(and for the DAML imports - it'd be nice to say explicitly "You don't need to change anything in your DAML code)

Reply:
More nuanced than that - I did fix up some instances, and they were just replacing `GHC.Tuple` with `DA.Tuple`, so I believe if my instructions are interpreted as a list of replacements that's sufficient, at least for our test suite. What I don't know is if there will be more subtle impacts or other places that need tweaks. I don't think so. I also can't say whether it's in import statements or usages, because that depends on what users have done.

Reply:
Cleaned it up a bit

Reply:
I guess this is what I'd suggest, then:

```
- **BREAKING CHANGE - DAML Standard Library**: Moved the ``Tuple`` and ``Either`` types to ``daml-prim:DA.Types`` rather than exposing internal locations.

  You don't need to change DAML code as a result of this change. However if you're using the Java or Scala bindings, you need to migrate by replacing usages or imports of ``GHC.Tuple`` and ``DA.Internal.Prelude`` with ``DA.Tuple``.
```

Reply:
I went with that.

Reply:
I had a quick chat with @gerolf-da:
- People using the Java/Scala codegen need to replace `import ghc.tuple.*` with `import da.types.*`. Same for `DA.Internal.Prelude`.
- People using the Ledger API directly need to do the change mentioned above.

Reply:
Thanks both, I've integrated that update into the notes.